lib/Partitioner/Partitioner.cpp

```diff
@@ -328,6 +389,25 @@ void Partitioner::initOpComputeTime(Function *F) {
       totalOps *= (inputChannels * 1.0 / nGroups);
       break;
     }
+#ifdef GLOW_WITH_HABANA
```

Comment:
Is this needed? HabanaConvolutionNodeKind will be defined regardless right?

Reply:
I’m not sure - I think I saw errors without this. But perhaps someone can tell if it’s possible to remove this in some other way?

Reply:
I think HabanaConvolutionNode is only defined if `GLOW_WITH_HABANA` is enabled since the backend-specific parts of NodeGen are guarded by `ifdef`s and CMake variables.